src/mono/wasm/runtime/loader/run.ts

```diff
@@ -308,7 +308,7 @@ export class HostBuilder implements DotnetHostBuilder {
                 throw new Error("URLSearchParams is supported");
             }
 
-            const params = new URLSearchParams(window.location.search);
+            const params = new URLSearchParams(globalThis.window.location.search);
```

Comment:
I think this is redundant, but I'm fine with it

Reply:
it would not even parse in strict es6, complaining about undefined symbol